vm/src/obj/objslice.rs

```diff
@@ -1,18 +1,19 @@
-use num_bigint::BigInt;
-
 use crate::function::PyFuncArgs;
-use crate::pyobject::{PyContext, PyObjectRef, PyRef, PyResult, PyValue, TypeProtocol};
+use crate::pyobject::{IdProtocol, PyContext, PyObjectRef, PyRef, PyResult, PyValue, TypeProtocol};
 use crate::vm::VirtualMachine;
 
-use super::objint;
-use crate::obj::objtype::PyClassRef;
+use crate::obj::objint::PyInt;
+use crate::obj::objtype::{class_has_attr, PyClassRef};
+use num_bigint::BigInt;
 
-#[derive(Debug)]
+#[derive(Debug, FromArgs)]
```

Comment:
Built-in types like `PySlice` shouldn't be implementing `FromArgs`

Reply:
I just thought it was convenient to do so, but now, I realize that it's s bad idea as any functions that accept PySlice would try and use that FromArgs implementation.